dashboard/apps/web/littlehorse-public-api/acls.ts

```diff
@@ -698,7 +698,7 @@ export const PutPrincipalRequest = {
     if (message.globalAcls !== undefined) {
       ServerACLs.encode(message.globalAcls, writer.uint32(26).fork()).ldelim();
     }
-    if (message.overwrite === true) {
+    if (message.overwrite !== false) {
```

Comment:
I'm not sure why `compile-proto` is changing this

Reply:
Yeah, it might be worth looking.